includes/api/class-wc-rest-coupons-controller.php

```diff
@@ -149,14 +141,57 @@ public function prepare_object_for_response( $object, $request ) {
 
 		// Format date values.
 		foreach ( $format_date as $key ) {
-			$data[ $key ] = $data[ $key ] ? wc_rest_prepare_date_response( get_gmt_from_date( date( 'Y-m-d H:i:s', $data[ $key ] ) ) ) : null;
+			$datetime 	  			= $data[ $key ];
+			$data[ $key ] 			= wc_rest_prepare_date_response( $datetime, false );
+			$data[ $key . '_gmt' ] 	= wc_rest_prepare_date_response( $datetime );
 		}
 
 		// Format null values.
 		foreach ( $format_null as $key ) {
 			$data[ $key ] = $data[ $key ] ? $data[ $key ] : null;
 		}
 
+		return array (
+			'id' => $object->get_id(),
+			'code' => $data['code'],
+			'amount' => $data['amount'],
+			'date_created' => $data['date_created'],
+			'date_created_gmt' => $data['date_created_gmt'],
+			'date_modified' => $data['date_modified'],
+			'date_modified_gmt' => $data['date_modified_gmt'],
+			'discount_type' => $data['discount_type'],
+			'description' => $data['description'],
+			'date_expires' => $data['date_expires'],
+			'date_expires_gmt' => $data['date_expires_gmt'],
+			'usage_count' => $data['usage_count'],
+			'individual_use' => $data['individual_use'],
+			'product_ids' => $data['product_ids'],
+			'excluded_product_ids' => $data['excluded_product_ids'],
+			'usage_limit' => $data['usage_limit'],
+			'usage_limit_per_user' => $data['usage_limit_per_user'],
+			'limit_usage_to_x_items' => $data['limit_usage_to_x_items'],
+			'free_shipping' => $data['free_shipping'],
+			'product_categories' => $data['product_categories'],
+			'excluded_product_categories' => $data['excluded_product_categories'],
+			'exclude_sale_items' => $data['exclude_sale_items'],
+			'minimum_amount' => $data['minimum_amount'],
+			'maximum_amount' => $data['maximum_amount'],
+			'email_restrictions' => $data['email_restrictions'],
+			'used_by' => $data['used_by'],
+			'meta_data' => $data['meta_data'],
```

Comment:
Instructions here: https://www.granneman.com/webdev/editors/sublime-text/packages/how-to-install-and-use-sublime-alignment/